tests/unit/lc/conftest.py

```
@@ -0,0 +1,201 @@
+# tests/unit/lc/conftest.py
+"""
+LangChain-specific test configuration.
+
+This conftest handles:
+- Safe imports for LangChain dependencies
+- Automatic test skipping if LangChain not available
+- Mock fixtures for LangChain components
+- Environment variables for LangChain testing
+"""
+import os
+import tempfile
```

Ruff (0.11.9)

`tempfile` imported but unused

Remove unused import: `tempfile`

(F401)